articles/active-directory/develop/msal-net-client-assertions.md

```diff
@@ -74,33 +74,40 @@ sub | {ClientID} | The "sub" (subject) claim identifies the subject of the JWT,
 Here is an example of how to craft these claims:
 
 ```csharp
-private static IDictionary<string, string> GetClaims()
+using System.Collections.Generic;
+private static IDictionary<string, object> GetClaims(string tenantId, string clientId)
 {
-      //aud = https://login.microsoftonline.com/ + Tenant ID + /v2.0
-      string aud = $"https://login.microsoftonline.com/{tenantId}/v2.0";
-
-      string ConfidentialClientID = "00000000-0000-0000-0000-000000000000" //client id
-      const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
-      DateTime validFrom = DateTime.UtcNow;
-      var nbf = ConvertToTimeT(validFrom);
-      var exp = ConvertToTimeT(validFrom + TimeSpan.FromSeconds(JwtToAadLifetimeInSeconds));
-
-      return new Dictionary<string, string>()
-           {
-                { "aud", aud },
-                { "exp", exp.ToString() },
-                { "iss", ConfidentialClientID },
-                { "jti", Guid.NewGuid().ToString() },
-                { "nbf", nbf.ToString() },
-                { "sub", ConfidentialClientID }
-            };
+    //aud = https://login.microsoftonline.com/ + Tenant ID + /v2.0
+    string aud = $"https://login.microsoftonline.com/{tenantId}/v2.0";
+
+    string ConfidentialClientID = clientId; //client id 00000000-0000-0000-0000-000000000000
+    const uint JwtToAadLifetimeInSeconds = 60 * 10; // Ten minutes
+    DateTimeOffset now = DateTimeOffset.UtcNow;
+    DateTimeOffset validFrom = now.AddSeconds(-15); // Clock skew
```

Comment:
I think the service will take care of clock skew, and it's probably more generous than 15s.